x-pack/plugins/security_solution/server/lib/detection_engine/scripts/rules/queries/query_with_threat_mapping.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Query with a threat mapping",
+  "name": "Query with a indicator mapping",
```

Comment:
Super duper nit: 
```suggestion
  "name": "Query with an indicator mapping",
```

Reply:
Crap missed this after the merge! Thanks though, this shows the level you're looking at the code. I appreciate it. Always nervous of making simple slip ups in the UI.